README.md

```diff
@@ -1,9 +1,8 @@
 # Karoo Array Telescope GPU-Accelerated Correlator/Beamformer (katgpucbf)
 
-This repository implements both F- and X-engines for a GPU-based correlator,
-developed for the MeerKAT Extension Radio Telescope by the South African Radio
-Astronomy Observatory (SARAO). A B-engine for beamforming is planned, but not
-yet included.
+This repository implements both F-, X-, and B-engines for a GPU-based correlator,
```

Comment:
My only nit-pick is that "both" applies when you list only two things (as far as I understand it anyway).

```suggestion
This repository implements F-, X-, and B-engines for a GPU-based correlator,
```

Reply:
Thanks, I'd missed that when I added B-engines.